src/wazuh_db/wdb_rootcheck.c

```diff
@@ -11,8 +11,6 @@
 
 #include "wdb.h"
 
-static const char *SQL_INSERT_PM = "INSERT INTO pm_event (date_first, date_last, log, pci_dss, cis) VALUES (?, ?, ?, ?, ?);";
-static const char *SQL_UPDATE_PM = "UPDATE pm_event SET date_last = ? WHERE log = ?;";
 static const char *SQL_DELETE_PM = "DELETE FROM pm_event;";
```

Comment:
You can remove this query and use `WDB_STMT_ROOTCHECK_DELETE_PM`

Reply:
Cannot due that, since `SQL_STMT` is static.